evalml/tests/integration_tests/test_data_checks_and_actions_integration.py

```diff
@@ -51,7 +51,7 @@ def test_data_checks_suggests_drop_cols():
         }
     )
     y = pd.Series([1, 0, 0, 1, 1])
-    data_check = HighlyNullDataCheck()
+    data_check = NullDataCheck()
```

Comment:
For more clarity: 

If we were to put a debugger in, `actions` would look something like `[{'code': 'DROP_COL', 'data_check_name': 'NullDataCheck', 'metadata': {'columns': ['all_null'], 'rows': None, 'parameters': {}}}, {'code': 'IMPUTE_COL', 'data_check_name': 'NullDataCheck', 'metadata': {'columns': ['lots_of_null'], 'rows': None, 'is_target': False, 'parameters': {'impute_strategies': {'lots_of_null': {'impute_strategy': 'mean'}}}}}]` which has the imputation information we want!

It's `make_pipeline_from_actions` which won't actually convert that action to anything useful in the pipeline right now, since it's not sure what to do with a `IMPUTE_COL` action where `is_target` is False (we currently support just target imputation). It's coming up though!

Reply:
@angela97lin Thanks for the context! Yep I agree the actions are returning the right thing (I'm still not sure about `mode` though) I just thought it was in-scope to modify `make_pipeline_from_actions` in this PR. I'll wait for #3237 !

Reply:
@freddyaboulton Oops totally missed the mode comment, you're right 😝 Let me update that to `most_frequent`!